.gitlab-ci.yml

```diff
@@ -39,12 +39,14 @@ variables:
           $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs and from web interface
     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
 
-# Deploy on production goes only on tag
+# Deploy on production goes only manually
 .deploy-prod-refs:                 &deploy-prod-refs
   rules:
     - if: $CI_PIPELINE_SOURCE == "web" &&
```

Comment:
Can't this rule be removed in favor of `$CI_COMMIT_REF_NAME == "master"` alone? It's the same idea behind https://gitlab.parity.io/parity/opstooling/vanity-service/-/merge_requests/46#note_157369